tests/test_render.py

```diff
@@ -1352,6 +1352,10 @@ def update_body(chapter):
             Output('button-output', 'children'),
             [Input('button', 'n_clicks')])
         def this_callback_takes_forever(n_clicks):
+            if not n_clicks:
+                # initial value is quick, only new value is slow
+                # also don't let the initial value increment call_counts
+                return 'Initial Value'
```

Comment:
I don't understand how this test was passing on master - it failed for me locally, and failed in this branch on ci, until this fix.